docs/api/cookies.md

```diff
@@ -120,3 +120,7 @@ Removes the cookies matching `url` and `name`
 Returns `Promise<void>` - A promise which resolves when the cookie store has been flushed
 
 Writes any unwritten cookies data to disk.
+
+Cookies written by any method will not be written to disk immediately, but will be written every 30 seconds or when the number of cookies exceeds 512.
+
+Calling this method can cause the cookie to be written to disk immediately
```

Comment:
@codebytere I did some sleuthing and found more context buried in Chrome docs and comments.

From Chromium's [Cookie docs](https://chromium.googlesource.com/chromium/src/+/HEAD/net/cookies/README.md):

> The `SQLitePersistentCookieStore` is the main implementation of `CookieMonster::PersistentCookieStore`. It stores cookies in a SQLite database on disk, at a filepath specified by the user's profile.

Then, the [implementation of `SQLitePersistentCookieStore`](
https://chromium.googlesource.com/chromium/src/net/+/master/extras/sqlite/sqlite_persistent_cookie_store.cc#251) has a useful comment that says:

> Subsequent to loading, mutations may be queued by any thread using AddCookie, UpdateCookieAccessTime, and DeleteCookie. These are flushed to disk on the BG runner every 30 seconds, 512 operations, or call to Flush(), whichever occurs first.

-----

As for this PR itself: I think it would be helpful to put this context on writing cookies to disk outside of the API documentation for the `flushStore()` method. 
Perhaps at the top of this doc or in a separate tutorial.

I think this PR has value because we expose an API that "writes any unwritten cookies data to disk", but don't really explain where/when/how cookies are written on disk in Electron elsewhere.

I'd like for our users to be able to understand that by glancing at our documentation rather than digging through Chrome code.

Reply:
Yes, that's my idea, and that's where the documentation comes in